pyccel/ast/builtin_methods/list_methods.py

```diff
@@ -83,7 +84,7 @@ def append_argument(self):
         """
         return self._append_arg
 
-#==============================================================================
+#=======================================================================================
```

Comment:
Same comment

Reply:
fixed in 12cd0b2a08f4f7d59e7c61e94769a7a9d9092039